src/systemjs-production-loader.js

```diff
@@ -315,12 +309,12 @@ function coreInstantiate (key, processAnonRegister) {
   }
 
   if (wasm)
-    return instantiateIfWasm(key)
+    return instantiateIfWasm(this, key)
     .then(function (sourceOrModule) {
-      if (typeof sourceOrModule !== 'string')
-        return sourceOrModule;
```

Comment:
This used to return, but my code doesn't because I can't figure out _what_ it would return other than `undefined`?